FreshBuy/pages/wishlist/wishlist.html

```diff
@@ -0,0 +1,144 @@
+                    <button class="search-icon"><i class="fa fa-search"></i></button>
+                    <input type="text" name="username" placeholder="Search here...">
+                </div>
+            </section>
+            <ul class="list-style-none account-data">
+                <li class="list-inline-item">
+                    <a href="../authentication/login.html"class="btn btn-solid-primary link-btn">Login</a>
+                </li>
+                <li class="list-inline-item">
+                    <div class="avatar avatar-text-xs avatar-text img-round user-profile" role="img" alt="Avatar">AB</div>
+                </li>
+                <li class="list-inline-item">
+                    <button class="badge-container badge-btn">
+                        <a href="../wishlist/wishlist.html">
+                            <i class="fas fa-heart icon"></i>
+                            <span class="status-badge-icon bg-red">5</span>
+                        </a> 
+                    </button>
+                </li>
+                <li class="list-inline-item">
+                    <button class="badge-container badge-btn">
+                        <a href="../cart/cart.html">
+                            <i class="fas fa-cart-shopping"></i>
+                            <span class="status-badge-icon bg-red">3</span> 
+                        </a>
+                    </button>
+                </li>
+                <li class="list-inline-item">
+                    <button class="btn btn-icon-primary user-signout">
+                        <a href="../../../index.html">
+                            <i class="fa fa-sign-out fa-2x"></i>
+                        </a>
+                    </button>
+                </li>
+            </ul>
+        </nav>
+        <div class="wishlist-products">
+            <p class="product-page-heading text-3xl text-bold"><span class="no-items-in-wishlist">5</span> Products in your wishlist<p>
+            <div class="wishlist-items">
+                <div class="card-container vertical">
+                    <div class="card-img vertical-img border-bottom">
+                        <img src="../../assets/images/fruits/apple.png" alt="Apple">
+                    </div>
+                    <div class="card-details card-details-vertical">
+                        <h2 class="card-heading">Apple<span><i class="fas fa-heart wishlisted"></i></span></h2>
+                        <p class="card-sub-heading">Fresh apple from farm</p>
+                        <div class="rating text-lg">
+                            <span class="rating-value">4.3<i class="fa fa-star checked margin-l"></i></span>
+                            (<span class="rating-number">2333</span>)
+                        </div>
+                        <h4 class="product-price">Rs.120 <span class="original-price text-strike-through">Rs.150</span><span class="discount-percentage">20% off</span></h4>
```

Comment:
You can use rupee symbol to make it look good

Reply:
Thank you for the suggetion